tools/java_api/src/main/java/com/kuzudb/DataType.java

```diff
@@ -30,19 +30,19 @@ private void checkNotDestroyed() throws ObjectRefDestroyedException {
     }
 
     /**
-    * Finalize.
+    * Close the connection and release the underlying resources. This method is invoked automatically on objects managed by the try-with-resources statement.
```

Comment:
My example is for `Connection` class. You should change it for each class and not blindly copy it everywhere.

Reply:
Yes, sorry didn't check the comment closely enough, will fix